### PR TITLE
type: export AffixRef interface

### LINKED ISSUE #47979: AffixRef等这类ref用到的类型，为什么没有导出呢？
### What problem does this feature solve?

1. 使用 useRef<AffixRef>()时，可以获得类型提示，目前AffixRef没有从antd导出
2. 类似情况，不止AffixRef一个

### What does the proposed API look like?

import { AffixRef } from 'antd'

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [x] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

- close #47979

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | type: export AffixRef interface |
| 🇨🇳 Chinese | type: 导出 AffixRef 类型 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/affix/index.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ interface AffixState {
   prevTarget: Window | HTMLElement | null;
 }
 
-interface AffixRef {
+export interface AffixRef {
```
Comment (MadCcc) [inline components/affix/index.tsx:56]: 咦我们是不是没有在文档里透出过这个方法

Comment (li-jia-nan) [reply components/affix/index.tsx:56]: 是的

Comment (sunven) [reply components/affix/index.tsx:56]: 都改了吧，不止这一个哦

Comment (li-jia-nan) [reply components/affix/index.tsx:56]: 不是所有类型都有必要透出吧

Comment (sunven) [reply components/affix/index.tsx:56]: 我觉得应该都要透出

Comment (li-jia-nan) [reply components/affix/index.tsx:56]: - #47983

**Review by MadCcc — APPROVED** [review]

**Pull request merged**